stdlib/public/core/DictionaryStorage.swift

```diff
@@ -118,7 +118,7 @@ internal class _EmptyDictionarySingleton: _RawDictionaryStorage {
   }
 
 #if _runtime(_ObjC)
-  @objc
+  @objc @usableFromInline
```

Comment:
Oh - these are correct but unfortunate: these are Objective-C entry points and they aren’t supposed to be visible (or used) as Swift methods. 

We currently have `_NSDictionaryCore` and the other shadow protocols as public. I think we should make them internal and remove these usableFromInlines in a followup PR. Cc @airspeedswift, @Catfish-Man